sdk/tvOS/OoyalaTVSkinSDK/Widgets/OOOoyalaTVBottomBars.h

```diff
@@ -10,7 +10,8 @@
 
 @interface OOOoyalaTVBottomBars : UIView
 
-- (id)initWithBackground:(UIView *)background;
+- (id)initWithBackground:(UIView *)background
```

Comment:
Please, prefer `instancetype` over `id` for `init`

Reply:
Fixed